### DISCUSSION TITLE
Change rem to em for more responsiveness

### DISCUSSION THREAD

**Discussion by anuraglol** [tailwindlabs/tailwindcss#5303] [Ideas]
Hey developers 👋
So it would be nice if tailwind css values are changed to em instead of rem

em is more responsive than rem

I know I can change it in config file, but still, it would be amazing

**Comment by trm217** [comment]
This most likely would cause more problems than it solves.
For example: You use `text-3xl` on a text and add a horizontal margin, to get some space to the element on the left.
Now you notice that the text is too large and you change to `text-2xl`, and instantly your layout starts breaking since you now have less margin on the left.
So depending on the font-size given on an element, the absolute values of all spacing based utility classes would change.
The DX would certainly decrease with that change.
